index.html

```diff
@@ -55,97 +55,8 @@ <h2 class="secondtitle">I'm a software developer</h2>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-        <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <div class="card"></div>
```

Comment:
Hi, the button to close the popup should be on the right side, as directed by the previous reviewer, kindly fix it. Also, it would be better if you would add some padding to the individual technology you have mentioned for said project to make it look more like Figma design
Here is the snapshot of your current application
![image](https://user-images.githubusercontent.com/33268927/175464424-562f10a2-ce91-49ee-a181-b59ca72209cd.png)